azurerm_container_group/vars.tf

```
@@ -0,0 +1,107 @@
+# General Variables
+
+variable "global_prefix" {
+  type = string
+}
+
+variable "environment" {
+  type = string
+}
+
+variable "region" {
+  type = string
+}
+
+variable "name" {
+  type = string
+}
+
+// Resource Group 
+variable "resource_group_name" {
+  type = string
+}
+
+// Storage Account
+variable "storage_account_name" {
+  type = string
+}
+
+variable "account_kind" {
```

```suggestion
variable "storage_account_kind" {
```

Same for the other variables